exercises.ipynb

```diff
@@ -52,59 +56,1032 @@
    },
    "outputs": [],
    "source": [
-    "def get_titles(soup):    \n",
-    "    \"\"\" Returns a list of titles on the page \"\"\"\n",
-    "    # the \"soup\" parameter is of the type that is\n",
-    "    # returned by Beautiful Soup when it parses HTML.\n",
-    "    # The function should then use the object to\n",
-    "    # extract a list of titles (of the lego sets)\n",
-    "    #\n",
-    "    # Lookup the documentation for Beautiful Soup\n",
-    "    # Figure out how to select the text of the title\n",
-    "    # of each legoset. A title should look like: \n",
-    "    # \"10252: Volkswagen Beetle\"\n",
-    "    pass\n",
+    "def get_all_titles(url):    \n",
+    "    \n",
+    "    ##initialize this for the while statement\n",
+    "    last_page = False\n",
+    "    \n",
+    "    #put the first url in the list b/c while loop only gets next url\n",
+    "    url_list = [url]\n",
+    "\n",
+    "    while last_page == False:\n",
```

Comment:
Works fine, but consider splitting this up into (at least) two functions, one gets the links, the other gets the titles. It's different responsibility!